biopandas/pdb/pandas_pdb.py

```diff
@@ -864,3 +864,29 @@ def to_pdb_stream(self, records: tuple[str] = ("ATOM", "HETATM")) -> StringIO:
         output.write("\n")
         output.seek(0)
         return output
+
+    @staticmethod
+    def gyradius(df: pd.DataFrame) -> float:
+        """Compute the Radius of Gyration of a molecule
+
+        Parameters
+        ----------
+        df : pandas.DataFrame
+            DataFrame with ATOM entries
+
+        Returns
+        ---------
+        rg : float
+            Radius of Gyration of df in Angstrom
+
+            """
+        # could be made as a class variable if it will be needed elsewhere
+        atomic_masses = {"C": 12.0107, "O": 15.9994, "N": 14.0067, "S": 32.065}
+
+        coords = df[["x_coord", "y_coord", "z_coord"]].to_numpy()
+        masses = df["element_symbol"].map(lambda atom: atomic_masses.get(atom, 0)).to_numpy()
+        total_mass = masses.sum()
+        center_of_mass = (masses[:, None] * coords).sum(axis=0) / total_mass
+        distances = np.linalg.norm(coords - center_of_mass, axis=1)
+        rg = np.sqrt((distances ** 2 * masses).sum() / total_mass)
+        return round(rg, 4)
```

Comment:
I'm not sure we make a presumption about the accuracy people may want this at.

Reply:
So I could add an extra argument that defaults at let's say 3 or 4? Again, looked at `rmsd` which rounds to 4, I guess I could add the extra argument for it too.